### PR TITLE
remove derive-more

### PR DESCRIPTION
**Pull request by fakeshadow**
## PR Type

Refactor

## PR Checklist
Check your PR fulfills the following:

- [x] Format code with the latest stable rustfmt

## Overview

Remove `derive-more` dep.
Besides the deprecated resolver actor there is only one instance of use. It's managable with manual impl of `Debug` and `Display`

### CODE REVIEW

File: src/actors/resolver.rs [hunk 1]
```diff
@@ -121,25 +121,46 @@ impl Message for ConnectAddr {
...
+impl fmt::Debug for ResolverError {
+    fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Self::Resolver(s) => fmt.debug_tuple("Resolver").field(s).finish(),
+            Self::InvalidInput(s) => fmt.debug_tuple("InvalidInput").field(s).finish(),
+            Self::Timeout => fmt.debug_tuple("Timeout").finish(),
+            Self::IoError(e) => fmt.debug_tuple("IoError").field(e).finish(),
+        }
+    }
+}
```
Comment (robjtede) [inline src/actors/resolver.rs:147]: what's up with built-in Debug derive?

File: src/address/mod.rs [hunk 2]
```diff
@@ -1,7 +1,8 @@
 use std::hash::{Hash, Hasher};
-use std::{error, fmt};
-
-use derive_more::Display;
+use std::{
+    error,
+    fmt::{Debug, Display, Formatter, Result as FmtResult},
```
Comment (robjtede) [inline src/address/mod.rs:4]: nit: i prefer formatters to prefix fmt:: on usage

File: src/actors/resolver.rs (hunk 1 above)
Comment (fakeshadow) [reply src/actors/resolver.rs:147]: Other error types use manual Debug so I think it could be in consistent.

**Review by robjtede — APPROVED** [review]
LGTM

**Pull request merged**